lib/experimental/block-bindings/sources/index.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Define the mechanism to add new sources available in the block bindings API.
+ *
+ * @package gutenberg
+ */
+
+global $block_bindings_sources;
+$block_bindings_sources = array();
+if ( ! function_exists( 'register_block_bindings_source' ) ) {
+	/**
+	 * Function to register a new source.
+	 *
+	 * @param string   $source_name The name of the source.
+	 * @param function $source_args List of arguments for the block bindings source:
+	 *                              - label: The label of the source.
+	 *                              - apply: The callback executed when the source is processed in the server.
```

Comment:
The description for `apply` could be improved a bit, as this is probably the most important part of the API, and also the hardest for API users to understand.

Some things they'll need to know:
- When is this called?
- What are the params?
- What should be returned?

Reply:
Makes sense 🙂  Let's improve this part as well.

Reply:
It seems like a low hanging fruit so let’s address before landing this PR.

Reply:
Ok I updated the comments. I wasn't sure how to handle clarifying comments for code that isn't currently being used and/or is also temporary, so I updated the callback and did the best I could.